tests/src/JIT/HardwareIntrinsics/X86/General/VectorArgs.cs

```diff
@@ -79,7 +79,7 @@ unsafe static int Main()
 
         if (Sse41.IsSupported)
         {
-            Vector128<float> rgb = Sse.SetVector128(0, 1, 2, 3);
+            Vector128<float> rgb = Vector128.Create(3f, 2f, 1f, 0f);
```

Comment:
The `Create` methods take args from lowest index to highest index; which is opposite what `SetVector` did